nautobot/core/utils/requests.py

```diff
@@ -164,23 +165,38 @@ def normalize_querydict(querydict, form_class=None, filterset=None):
 
     Raises:
         AttributeError: If both `form_class` and `filterset` are provided.
+        AttributeError: Provided `form_class` is a subclass of `BulkEditForm` but `model` is not provided.
```

Comment:
Any chance of this being a breaking change for Apps?

Reply:
There is a chance but I think it is highly unlikely. The most likely scenario is that `send_bulk_edit_objects_to_job` is overridden for some reason, but that is a relatively new method that I do not think apps had the time to yet.

Reply:
Well, I guess unit test failure suggests that it might be a breaking change still.